libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -1029,8 +1029,8 @@ void AC_AutoTune_Multi::updating_angle_p_down(float &tune_p, float tune_p_min, f
 // P is increased until we achieve our target within a reasonable time
 void AC_AutoTune_Multi::updating_angle_p_up(float &tune_p, float tune_p_max, float tune_p_step_ratio, float angle_target, float meas_angle_max, float meas_rate_min, float meas_rate_max)
 {
-    if ((meas_angle_max > angle_target*(1+0.5f*aggressiveness)) ||
-        ((meas_angle_max > angle_target) && (meas_rate_min < -meas_rate_max*aggressiveness))) {
+    if ((meas_angle_max > angle_target * (1 + 0.5 * aggressiveness)) ||
```

Comment:
1.0 like L945

Reply:
That looks like it might make for another method, perhaps, if it's supposed to be the same in all cases?